tests/utils.ts

```diff
@@ -1,6 +1,11 @@
 import { Locator, Page } from '@playwright/test';
 import shortid from 'shortid';
 
+export enum TableTypes {
```

Comment:
Made a new enum since I think the other e2e tests that I need to write will need different types of tables to fully test their functionality. If this ends up being a small enum list I will just move over the unique command strings into table.spec.ts instead

Reply:
Yes looks fine. We could also do something like import the template string from a file name, but then we'd need to search/replace for the table name, I think this is decent enough.